src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/AesCcm.Windows.cs

```diff
@@ -25,7 +24,7 @@ private void ImportKey(ReadOnlySpan<byte> key)
             Span<byte> tag,
             ReadOnlySpan<byte> associatedData = default)
         {
-            AesAEAD.Encrypt(s_aesCcm, _keyHandle, nonce, associatedData, plaintext, ciphertext, tag);
+            AEADCommon.Encrypt(_keyHandle, nonce, associatedData, plaintext, ciphertext, tag);
```

Comment:
Wow, this file tells me when my brain decides something has too many caps.  `AEADCommon` (5) was OK.  `AEADBCrypt` (6) was not :smile:.

Reply:
My brain had the same problem. It's hardwired to recognize `DB` in `AEA-DB-Crypt`.

Reply:
FDG naming conventions it is, then!